plugins/hipchat.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * The Hipchat integration plugin
+ * @module Hipchat
+ */
+"use strict";
+
+var HipchatApi = require('hipchatter');
+
+/**
+ * @class Hipchat
+ * @param config {Object} The plugins configs
+ * @param mergeatron {Mergeatron} An instance of the main Mergeatron object
+ * @constructor
+ */
+var Hipchat = function(config, mergeatron) {
+	this.config = config;
+	this.mergeatron = mergeatron;
+	this.api = new HipchatApi(this.config.token);
+};
+
+/**
+ * Translates a github user to a hipchat user
+ *
+ * @method getUser
+ * @param githubUser {String}
+ * @return {String}
+ */
+Hipchat.prototype.getUser = function(githubUser) {
+	if (this.config.github_to_hipchat_users && this.config.github_to_hipchat_users[githubUser]) {
+		return this.config.github_to_hipchat_users[githubUser];
+	}
+	return githubUser;
```

Comment:
maybe we should consider the use case that a user doesn't want notifications. So maybe if you're not in this hash, then you won't get notifs, rather than trying to fall back on the github username which will most certainly cause a few errors in log.